### PR TITLE
Added \protect to tikz path in some icons 

### PR DESCRIPTION
**Pull request by gcorrad**
Fix broken compilation in classic style.
Related to line 73 in moderncvheadi.sty (the head of classic)

### CODE REVIEW

**Comment by mefuller** [comment]
That definitely appears to have done the trick for me, but I would like to get a second reviewer confirming this before merging.

**Review by mefuller — APPROVED** [review]
Seems to do exactly what's needed

**Review by ShadowMitia — APPROVED** [review]
I also confirm that without `\protect` the classic style doesn't compile.

**Pull request merged**
